### DISCUSSION TITLE
Very low performance with new RTX 4080

### DISCUSSION THREAD

**Discussion by DaWhigit** [bmaltais/kohya_ss#417] [Q&A]
Hey everyone, I just upgraded my GPU from a 3060 ti to a 4080 and I am getting really low it/s when running LoRA or Dreambooth something in the order of 1.4 it/s. I am also showing 1% GPU usage.

Current build:
Windows 10 pro
Asus TUF H570 Pro Wifi
Intel 11700k
Corsair vengeance pro 64GB 3600MHz
Asus TUF RTX 4080 OC
nvidia driver 531.29
1TB Samsung 970 evo plus M.2 SSD
2TB Samsung 970 evo plus M.2 SSD
Corsair RM 1000x PSU
And tons of cooling.

I have tried updating the .dll files, uninstalling and reinstalling the GPU, switching out drivers all with no impact. Does anyone have any suggestions? I am attaching a screenshot of my performance metrics while running LoRA.

![image](https://user-images.githubusercontent.com/128553030/226763262-60d54d5c-f93d-4a81-99f6-ff2b9afaccd4.png)

If anyone else has encountered similar issues, any potential fix would be very appreciated.

***EDIT
For comparison's sake I upgraded to windows 11. Now running at 2.2 or so it/s. A significant improvement. Should I be getting more still?

**Comment by bananasss00** [comment]
Similar problem on the rtx 4080. 
But my speed on windows 10 is faster than on windows 11.

I specifically installed windows 10 and windows 11 from scratch, ran lora training on both.

Windows 10 = ~4it/sec
Windows 11 = ~2.7-3it/sec

Tried disabling / enabling xformers, updating cudnn for torch, changing precision in accelerate config: 'no', 'fp16', 'bf16'. Nothing helped to raise the speed above 3it/sec on Windows 11. What could be the problem?

**Comment by bananasss00** [comment]
Fix slow speed in windows 11 here: https://github.com/bmaltais/kohya_ss/issues/1337#issuecomment-1668773013
